.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
You should always start and end your document with markers, whether you use directives or not. It's a document, so it should have document markers.